.changeset/hip-turtles-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/database': patch
+---
+
+RTDB query gets shouldn't send server requests for listened-to data
```

Comment:
This might need a little rewrite. Maybe something like

`A `get()` issued for a query that is already being listened to no longer sends a request to the backend."